PrebidMobile/AdUnits/BannerAdUnit.swift

```diff
@@ -15,16 +15,34 @@
 
 import UIKit
 
-public class BannerAdUnit: BannerBaseAdUnit {
+@objcMembers
+public class BannerAdUnit: BannerBaseAdUnit, BannerBasedAdUnitProtocol, VideoBasedAdUnitProtocol {
+    
+    public var bannerParameters: BannerParameters {
+        get { adUnitConfig.adConfiguration.bannerParameters }
+        set { adUnitConfig.adConfiguration.bannerParameters = newValue }
+    }
+    
+    public var videoParameters: VideoParameters {
+        get { adUnitConfig.adConfiguration.videoParameters }
+        set { adUnitConfig.adConfiguration.videoParameters = newValue }
+    }
+    
+    public var adFormats: Set<AdFormat> {
+        get { adUnitConfig.adFormats }
+        set { adUnitConfig.adFormats = newValue }
+    }
     
     public init(configId: String, size: CGSize) {
         super.init(configId: configId, size: size)
```

Comment:
Can we make the `adFormats` an obligatory property of AdUnit initializer so we are sure that this property will never be undefined?